src/components/Title.tsx

```diff
@@ -0,0 +1,89 @@
+import { Stack, SxProps, Typography } from '@mui/material';
+import { colorPalette } from '../theme/hugo/colors';
+
+type TitleVariant =
+  | 'globalXXS'
+  | 'globalXS'
+  | 'globalS'
+  | 'globalM'
+  | 'globalL'
+  | 'globalXL'
+  | 'globalXXL';
```

Comment:
```suggestion
  | 'XL'
  | 'L'
  | 'M'
  | 'S';
```

Acá haría algo así, porque dice que XS es solo para mobile